angular/src/shared/pages/create-candidate/current-requisition/current-requisition.component.ts

```diff
@@ -330,18 +351,66 @@ export class CurrentRequisitionComponent extends AppComponentBase implements OnI
   }
 
   onRequestStatusChange(id: number) {
-    const hasRequired = this.applyResultForm.get('onboardDate').hasValidator(Validators.required);
-    if (id !== REQUEST_CV_STATUS.Onboarded && hasRequired) {
-      this.applyResultForm.get('onboardDate').removeValidators(Validators.required);
-      this.applyResultForm.get('onboardDate').updateValueAndValidity();
-      return;
-    }
-    if (id === REQUEST_CV_STATUS.Onboarded) {
-      this.applyResultForm.get('onboardDate').setValidators([Validators.required]);
-      this.applyResultForm.get('onboardDate').updateValueAndValidity();
+    this.ValidateRequestCVLevel(id)
+    this.ValidateStaffRequestCVStatus(id);
+  }
+
+  ValidateRequestCVLevel(id: number)
```

Comment:
function name convention `ValidateRequestCVLevel` and `ValidateStaffRequestCVStatus`